Kubeadm_Installation_Scripts_and_Documentation/Kubeadm_Installation_Slave_Using_Containerd.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#Execute on ALL of your Worker Nodes
+
+#1. Perform pre-flight checks and reset the node:
+```bash
+sudo kubeadm reset -f
+```
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)